src/interfaces/requestsPublishers.js

```diff
@@ -80,8 +79,8 @@ export default function () {
 
   async function updateRequest(db, id, doc, user) {
     try {
-      if (validateDoc(doc, 'PublisherRequestContent')) {
-        if (hasPermission(user, 'publisherRequests', 'updateRequest')) {
+      if (hasPermission(user, 'publisherRequests', 'updateRequest')) {
+        if (validateDoc(doc, 'PublisherRequestContent')) {
           // Set backgroundProcessingState based on earlier if not re-defined in request
           const {backgroundProcessingState} = await publisherRequestsInterface.read(db, id, {});
           const newDoc = {...doc, backgroundProcessingState: doc.backgroundProcessingState ? doc.backgroundProcessingState : backgroundProcessingState};
```

Comment:
Can it be rewritten like this?
```suggestion
          const newDoc = {...doc, backgroundProcessingState: doc.backgroundProcessingState || backgroundProcessingState};
```

Reply:
Should be ok, but keeping the coding style consistent with previously written API styling.